crates/bevy_editor/src/main.rs

```diff
@@ -12,8 +12,10 @@
 //! - Finally, it will be a standalone application that communicates with a running Bevy game via the Bevy Remote Protocol.
 
 use bevy::prelude::*;
+use bevy_editor_camera::*;
 
 fn main() {
+    add(1, 2);
```

Comment:
remove the test func you did here

Reply:
Gotta keep it, otherwise we have unused imports (bevy_editor_camera)

Reply:
Or i can do a #[allow(...)]

Reply:
I suppose you can just remove the import as well since it's been tested and works now anyways. No need to keep it